src/web/app/components/sessions-recycle-bin-table/sessions-recycle-bin-table.component.html

```diff
@@ -91,5 +97,6 @@ <h2 class="text-muted">
         </table>
       </div>
     </div>
+    </div>
   </div>
```

Comment:
Nit on L99: Some alignment issue please check.

Currently:
![image](https://github.com/TEAMMATES/teammates/assets/46486515/445916b5-d1fb-4f6f-9d99-d014b63619ef)

Should be:
![image](https://github.com/TEAMMATES/teammates/assets/46486515/b7c5eabf-eaea-4725-a2ab-a4723a6ebc80)

Reply:
Fixed!